packages/perseus-core/src/widgets/image/index.ts

```diff
@@ -0,0 +1,31 @@
+import type {PerseusImageWidgetOptions} from "../../data-schema";
+import type {WidgetLogic} from "../logic-export.types";
+
+export type ImageDefaultWidgetOptions = Pick<
+    PerseusImageWidgetOptions,
+    "title" | "range" | "box" | "backgroundImage" | "labels" | "alt" | "caption"
+>;
+
+const defaultBoxSize = 400;
+const defaultRange = [0, 10] as any;
+
+const defaultWidgetOptions: ImageDefaultWidgetOptions = {
+    title: "",
+    range: [defaultRange, defaultRange],
+    box: [defaultBoxSize, defaultBoxSize],
```

Comment:
I suspect this is how it was defined originally, but it would be much easier to read this if these weren't variables.